AFQ/api/group.py

```diff
@@ -916,6 +916,89 @@ def assemble_AFQ_browser(self, output_path=None, metadata=None,
             sublink=page_subtitle_link)
 
 
+class ParallelGroupAFQ(GroupAFQ):
+    def __init__(self, *args, **kwargs):
+        orig = GroupAFQ(*args, **kwargs)
+
+        if "submitter_params" not in orig.parallel_params:
+            orig.parallel_params["submitter_params"] = {"plugin": "cf"}
+
+        if "cache_dir" not in orig.parallel_params:
+            orig.parallel_params["cache_dir"] = None
+
+        self.parallel_params = orig.parallel_params
+        self.pAFQ_kwargs = [pAFQ.kwargs for pAFQ in orig.pAFQ_list]
+
+        # Rename kwargs and clear "bids_info" and "base_fname"
+        # ParticipantAFQ takes in these parameters under one name but stores
+        # and uses them under another
```

Comment:
This is awkward. Could we change this in the implementation of `ParticipantAFQ`, instead of implementing all this renaming and deleting?